client/main.lua

```diff
@@ -388,7 +388,7 @@ RegisterNUICallback('selectItem', function(inData, cb)
         if action then
             action(data)
         elseif data.type == 'client' then
-            TriggerEvent(data.event, data)
+            TriggerEvent(data.event, data.arg)
```

Comment:
Certain resources are already setup to just use the data, I've forgotten which ones, but there are a few and I think people may have copied that behaviour over

Reply:
Alright, I've been analyzing the the resource and a few others to see how the table data is used. No other resource is passing through arguments to the client event and then they tend to use singular events for specific items with a lot of repeating code.

A simple temporary fix would be a simple if statement checking if the field args exists as such:
```suggestion
        if data.arg then
            TriggerEvent(data.event, data.arg)
        else
            TriggerEvent(data.event, data)
        end
```

But I don't really see a way of how people are passing through event arguments for the time being.